test/dynamo/test_export.py

```diff
@@ -3742,6 +3731,7 @@ def forward(self, x):
 
         self.assertTrue(torch.allclose(m(x), gm(x)))
 
+    @unittest.expectedFailure
```

Comment:
What's going on here? IIRC this test is important for quantization for 2.1?

Reply:
I'll have a check and get back. It has some complicated errors as I remember.